app/jobs/api/issues_open_issue_job.rb

```diff
@@ -6,6 +6,7 @@ class IssuesOpenIssueJob < IssueEventJob
 
     def payload(params)
       {
+        pull_request: params[:pull_request],
```

Comment:
Correct, in `WebhooksController#publish_pull_request_event` we essentially "rename" the `pull_request` part of the payload as `issue`. So I added a simple flag in 77b26c34087a30fd54ca38ce46746720efc33ff7 to identify PRs again:

```diff
        message = HashWithIndifferentAccess.new(
+        :pull_request => true,
         :issue => payload[:pull_request],
         ...
```

This change just passed that value on through.